build-system/tasks/extension-helpers.js

```diff
@@ -746,7 +746,7 @@ async function getBentoBuildFilename(dir, name, mode, options) {
  * @return {string}
  */
 function generateBentoEntryPointSource(name, toExport, outputFilename) {
-  const bentoCePath = path.relative(
+  const bentoCePath = path.posix.relative(
```

Comment:
Looks like this will work. Approving in advance so you can merge once you're satisfied.

Reply:
On second thought, it won't. Imports need to be relative paths, so this should probably just be something like:
```js
const bentoCePath = '../../src/bento-ce'
```

Reply:
`path.posix.relative` returns a relative path. Should work now